certbot/certbot/interfaces.py

```diff
@@ -380,7 +380,8 @@ class IDisplay(zope.interface.Interface):
     """Generic display."""
     # see https://github.com/certbot/certbot/issues/3915
 
-    def notification(message, pause, wrap=True, force_interactive=False):
+    def notification(message, pause, wrap=True, force_interactive=False,
```

Comment:
Agh. If we add the option to the *interface*, I think that means we'll have to go 2.0 (because then implementers will either have to add an `unused_kwargs` argument like we do in non-interactive file display, or specifically add `decorate` to the method signature). Which we could. Or we could just not add it to the interface, and instead only add it to our own implementations. Which option sounds best to you?

Reply:
I like having a private flag in the implementation more, just to avoid the interface break.

Adding `IDisplay.print` could be an option too. Avoids touching existing signatures and also avoids a pointer which only exists for the sake of not typing out `pause=False`.

Reply:
As discussed in the meeting, let's add a private flag and then eventually [deprecate this whole thing](https://github.com/certbot/certbot/issues/8434) in some way.